spec/values/calculation/atan2.hrx

```diff
@@ -0,0 +1,204 @@
+<===> README.md
+Most of the same behavior tested for `calc()` applies to `atan2()`, but for
+terseness' sake isn't tested explicitly.
+
+<===>
+================================================================================
+<===> unitless/input.scss
+a {b: atan2(1, -10)}
+
+<===> unitless/output.css
+a {
+  b: 174.2894068625deg;
+}
+
+<===>
+================================================================================
+<===> compatible_units/input.scss
+a {b: atan2(1cm, -10mm)}
+
+<===> compatible_units/output.css
+a {
+  b: 135deg;
+}
+
+<===>
+================================================================================
+<===> operation/math/input.scss
+a {b: atan2(10 + 6, 8 - 1)}
+
+<===> operation/math/output.css
+a {
+  b: 66.3706222693deg;
+}
```

Comment:
As with abs(), all you need is one test to verify that the normal calculation simplification behavior happens—and it should probably include both an operation that can be simplified and something that can't.